test/disk_usage_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+require 'rack/test'
+
+require 'smart_proxy_pulp_plugin/pulp_plugin'
+require 'smart_proxy_pulp_plugin/disk_usage'
+
+class DiskUsageTest < Test::Unit::TestCase
+  def test_has_path_should_be_true
+    disk_test = ::PulpProxy::DiskUsage.new(:path => '/etc')
```

Comment:
I would suggest using some local dir, for example 'test' instead. Not a significant concern, but I'd rather not rely on presence of linux-specific dirs.